### PR TITLE
Add: a-tiny-js-world by andrewklmn

### PR DESCRIPTION
**Pull request by andrewklmn**
### a-tiny-js-world

[Demo](https://andrewklmn.github.io/a-tiny-JS-world/) |
[Code base](https://github.com/andrewklmn/a-tiny-JS-world/blob/master/index.js)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

**Review by madmaxWMFU — CHANGES_REQUESTED** [review]
Good work!
See some comments below

**Comment by madmaxWMFU** [comment]
This is not what I mean 😟
I wanted you to just replace `const catWoman = Object.assign({}, woman);` on `const catWoman = [...woman];`
That's all.

**Comment by madmaxWMFU** [comment]
And bring the rest back because it's only pre OOP

File: submissions/andrewklmn/a-tiny-js-world/index.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+const inhabitants = [
+  dog,
+  cat,
+  woman,
+  catWoman,
+  man,
+];
+
+const getFormatedOutput = (obj) => Object.entries(obj).map((a) => {
```
Comment (madmaxWMFU) [inline submissions/andrewklmn/a-tiny-js-world/index.js:68]: As for me, it is a hard function. But it is your choice. 
```
  const getFormatedOutput = ({species, name, gender, legs, hands, saying, friends}) =>  {
    if(typeof(friends) === 'object'){
        friends = friends.map(friend => friend.name).join(', ');
    } 

    return `${species}; ${name}; ${gender}; ${legs}; ${hands}; ${saying}; ${friends}`;
  }

```

**Review by madmaxWMFU — APPROVED** [review]

File: submissions/andrewklmn/a-tiny-js-world/index.js (hunk 1 above)
Comment (andrewklmn) [reply submissions/andrewklmn/a-tiny-js-world/index.js]: Such a cool solution!
I took it into my code.

**Review by OleksiyRudenko — APPROVED** [review]
Well done!

**Pull request merged**